torch/quantization/fx/quantize.py

```diff
@@ -928,7 +928,19 @@ def get_next_i(module, qparams):
         return GraphModule(observed_root, self.quantized_graph)
 
     def _find_matches(self, graph, modules, patterns):
-        match_map = {}  # node name -> (root_node, match_value?)
+        """
```

Comment:
same for this one, a brief description for what this function does would help understanding as well

Reply:
sure, I was going to add more details in future PRs, but we can add it here